lighthouse/helpers/plates.py

```diff
@@ -40,7 +40,9 @@ def add_cog_barcodes(samples: List[Dict[str, str]]) -> List[Dict[str, str]]:
     except requests.ConnectionError:
         raise requests.ConnectionError("Unable to access baracoda")
 
-    return samples
+    # return centre prefix
+    # TODO: I didn't know how else to get centre prefix?
```

Comment:
One option could be to move the centre prefix obtaining out of this method in a future refactor.